src/calculateRentalCost.js

```diff
@@ -3,8 +3,18 @@
  *
  * @return {number}
  */
+const RENT_PRICE = 40;
+const DISCOUNT_AFTER_DAY_7 = 50;
+const DISCOUNT_AFTER_DAY_3 = 20;
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rent = RENT_PRICE * days;
+
+  if (days >= 7) {
+    return rent -DISCOUNT_AFTER_DAY_7;
+  } else if (days >= 3) {
+    return rent - DISCOUNT_AFTER_DAY_3;
```

Comment:
You don't need to use `else if` after `return` in the `if` block. The code after `return` won't be executed anyway. So, you can simplify your code by removing `else` keyword. Here is how you can do it:

js
if (days >= 7) {
  return rent - DISCOUNT_AFTER_DAY_7;
}

if (days >= 3) {
  return rent - DISCOUNT_AFTER_DAY_3;
}

return rent;
```
This makes the code cleaner and easier to read.